docs/UserGuide.adoc

```diff
@@ -386,10 +386,24 @@ _{explain how the user can enable/disable data encryption}_
 
 == Frequently Asked Questions (FAQ)
 
+*Q*: Where can I find Java version `9`? +
+*A*: You can download Java version `9` link:https://www.oracle.com/technetwork/java/javase/downloads/java-archive-javase9-3934878.html[here]. We recommend installing Java SE Development Kit `9.0.4` on your computer.
+
+*Q*: (Windows Only) I followed the instructions in <<Quick Start>> but the GUI did appear? +
```

Comment:
Do you mean "the GUI did not appear?"

Reply:
Thank you for pointing it out. I will fix it.